plugins/search/providers/elastic_search/batch/KExportMediaEsearchEngine.php

```diff
@@ -128,12 +131,22 @@ protected function retrieveEntryCategories ($entryId)
 		
 		$categoryEntryResult = KBatchBase::$kClient->categoryEntry->listAction($categoryEntryFilter, $pager);
 		
-		$result = array();
 		foreach ($categoryEntryResult->objects as $categoryEntry)
 		{
 			$result[] = $categoryEntry->categoryId;
 		}
 		
 		return $result;
 	}
-}
\ No newline at end of file
+	/**
+	 * @param $timestamp
+	 * @param $data
+	 * @return false|string
+	 */
+	protected function makeTimeStampHumanReadableIfNeeded($timestamp, $data) {
```

Comment:
@TalDubovKaltura please change the method name to something less specific - formatTimestamp for instance. You also don't need to pass the entire data object into this method - the $data->options would suffice.

Reply:
About the options parameter, I absolutely agree with you and I changed it.
as for the name, I changed it but I do think that function names should be meaningful and specific as possible, it is a known best practice.  while I'm calling a method called 'formatTimestamp' as a developer I'm expecting to pass it a parameter of format type, otherwise, I will ask myself which formatting the method using and I will assume that it will be formatted anyway(that's why I used the 'IfNeeded' in the name).